benchmarks/NodeClassification/train.py

```diff
@@ -16,7 +16,8 @@
 from utils import generate_model, parse_args, Models_need_to_be_densed,\
                   load_config_file, check_multiple_split,\
                   EarlyStopping, set_seed, check_binary_classification,\
-                  eval_rocauc
+                  eval_rocauc, Datasets_need_to_be_undirected,\
+                  Models_using_adamW

```

Comment:
Remove Models_using_adamW. Create a separate train yaml file for LINKX to use AdamW. The original paper of MixHop did not use AdamW so there is no need to use AdamW for MixHop.

Reply:
The separate train yaml file could be named as `LINKX_train.yaml`

Reply:
I think they do use AdamW for Mixhop. I have read the code and check it. Maybe I have mistaken it? 
They did not specify that the Mixhop should not use adam in experiment/mixhop_exp.sh